qe_gluster_firewall.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This file should be named something like `firewall.tendlr_gluster.something.yml` :)